### PR TITLE
doc(ingestion/tableau): Fix rendering ingestion quickstart guide

### PR DESCRIPTION
**Pull request by mohdsiddique**

### CODE REVIEW

**Comment by jjoyce0510** [comment]
Please fix the bad image urls.

File: docs/quick-ingestion-guides/tableau/setup.md [hunk 1]
```diff
@@ -10,42 +10,41 @@ In order to configure ingestion from Tableau, you'll first have to enable Tablea
...
+         <img width="75%" alt="Actions Site Role" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/tableau/tableau-user-list.png"/>
+         </p>
 
       -  Change user role to `Site Administrator Explorer`
 
-      <p align="center">
-      <img width="75%" alt="tableau site role" src="https://raw.githubusercontent.com/datahub-project/static-assets-fork/main/imgs/guides/tableau/tableau-site-role.png"/>
-      </p> 
+         <p align="center">
+         <img width="75%" alt="tableau site role" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/tableau/tableau-site-role.png"/>
```
Comment (hsheth2) [inline docs/quick-ingestion-guides/tableau/setup.md:45]: looks like this image was not redacted

File: docs/quick-ingestion-guides/tableau/configuration.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ If you do not see the Ingestion tab, please contact your DataHub admin to grant
 6. Select Tableau
 
 <p align="center">
-  <img width="70%" alt="Select Tableau from the options" src="https://raw.githubusercontent.com/datahub-project/static-assets-fork/main/imgs/guides/tableau/tableau-new-ingestion-source.png"/>
+  <img width="70%" alt="Select Tableau from the options" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/tableau/tableau-new-ingestion-source.png"/>
```
Comment (hsheth2) [inline docs/quick-ingestion-guides/tableau/configuration.md:63]: looks like most of these image links are broken https://docs-website-1ua8gifbf-acryldata.vercel.app/docs/quick-ingestion-guides/powerbi/setup

Comment (mohdsiddique) [reply docs/quick-ingestion-guides/tableau/configuration.md:63]: Hi @hsheth2 
Please verify again, I have corrected the links, however whenever from git I am trying to download or open the image, I am getting below error 

image

The endpoint `raw.githubusercontent.com` is not rechable.

**Review by jjoyce0510 — APPROVED** [review]

**Pull request merged**